src/main/java/core/basesyntax/Counter.java

```diff
@@ -15,7 +15,9 @@ public Counter(int value) {
     public void decreaseValue() {
         logger.info(String.format(MESSAGE,
                 "Before decrementing", Thread.currentThread().getName(), value));
-        value--;
+        synchronized (this) {
+            value--;
+        }
```

Comment:
we should add logs to PR, and also make logs consistent - maybe let's synchronize logging operations as well?